grails-app/conf/application.yml

```diff
@@ -12,7 +12,10 @@ info:
         version: '@info.app.version@'
         grailsVersion: '@info.app.grailsVersion@'
 spring:
-
+    autoconfigure:
```

Comment:
Reason of the change:
Plugin initializes the same beans that Spring Boot's autoconfiguration is initializing which leads to duplicate beans created and app fails to start.